src/test/java/uk/gov/hmcts/reform/sandl/snlevents/controllers/HearingControllerTest.java

```diff
@@ -165,12 +168,16 @@ private Hearing createHearing() {
         h.setHearingType(new HearingType("code", "desc"));
         h.setCaseType(new CaseType("code", "desc"));
         h.setPriority(Priority.High);
+
+        val statusConfig = new StatusConfig();
```

Comment:
A side note, thanks to @Import({TestConfiguration.class, StatusesTestConfiguration.class})
you can @Autowire StatusConfigService or Repository with mocked Statuses and grab a proper one from there.
Why? because it gives you prepopulated all the values which can be used by manager.
Might be not needed now, but later?